tools/fiptool/fiptool.c

```diff
@@ -52,34 +52,6 @@
 #define OPT_PLAT_TOC_FLAGS 1
 #define OPT_ALIGN 2
 
-static image_desc_t *lookup_image_desc_from_uuid(const uuid_t *uuid);
```

Comment:
OK.  The last one has been replaced with a new one.
(I will plan to fix this issue and make it consistent, though.)